### PR TITLE
chaos: attribut notification_date added in the impact

### PR DESCRIPTION
**Pull request by kadhikari**
# Description

This PR adds an attribut notification_date in the impact

This is a new feature:
http://jira.canaltp.fr/browse/BOT-16

The branch  Feature_BOT-14_programmable_notification is a valid branch merged in the PR : https://github.com/CanalTP/Chaos/pull/230/

### CODE REVIEW

File: documentation/disruptions.md [hunk 1]
```diff
@@ -628,7 +628,8 @@ Retrieve one existing disruption:
                                 "end": "2014-05-22T02:15:00Z"
                             }
                         ],
-                        "send_notifications": true
+                        "send_notifications": true,
+                        "notification_date": "2014-04-31T17:00:00Z"
```
Comment (dvdn) [inline documentation/disruptions.md]: Is it mandatory ?

Comment (kadhikari) [inline documentation/disruptions.md]: notification_date is not mandatory. I will update document

Comment (kadhikari) [inline documentation/disruptions.md]: As discussed with @dvdn we will create a Ticket to add a table of attributs "required" for each api
Example:
Input Data:
Name     Description    Required
"..."         "........"             true

Comment (pthegner) [inline documentation/disruptions.md]: https://github.com/geocoders/geocodejson-spec/tree/master/draft
-> we could use it for comment required elements on json body

**Comment by dvdn** [comment]
TODO created : http://jira.canaltp.fr/browse/BOT-37

**Pull request merged**